capstone/src/main/webapp/styles/members.css

```diff
@@ -0,0 +1,123 @@
+#members-container {
+  background-color: #fefefe;
+  border-radius: 10%;
+  color:  #C4C4C4;
```

Comment:
same note about using the variables: `var(--near-white)` and `var(--badges-gray)`